packages/world-module-erc20/ts/exports/index.ts

```diff
@@ -0,0 +1,25 @@
+import { encodeAbiParameters, stringToHex } from "viem";
+
+export type DefineERC20ConfigInput = {
+  namespace: string;
+  name: string;
+  symbol: string;
+};
+
+export function defineERC20Config({ namespace, name, symbol }: DefineERC20ConfigInput) {
```

Comment:
how about `defineERC20Module` since it returns module input?

and can we specify the return type here so we know we're always getting config-compatible module input